cmd/kubeadm/app/apis/kubeadm/env.go

```diff
@@ -31,7 +33,7 @@ func SetEnvParams() *EnvParams {
 
 	envParams := map[string]string{
 		"kubernetes_dir":  "/etc/kubernetes",
-		"host_etcd_path":  "/var/lib/etcd",
+		"host_etcd_path":  v1alpha1.DefaultEtcdDataDir,
```

Comment:
Sorry for being unclear earlier, I don't think this is used anymore (if it is, switch to the new method)
It could/should be removed